src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Completion.cs

```diff
@@ -58,7 +58,7 @@ public class Command
         /// <summary>
         /// The identifier of the actual command handler, like `vsintellicode.completionItemSelected`.
         /// </summary>
-        public string CommandStr { get; set; }
+        public string command { get; set; }
```

Comment:
You have other places in the code that reference this value - make sure you're updating them (and verifying the tests still pass)

Reply:
Fixed the test.